bedrock/firefox/templates/firefox/welcome/page3.html

```diff
@@ -0,0 +1,106 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero with context %}
+{% from "macros.html" import fxa_cta_link with context %}
+
+{% add_lang_files "firefox/welcome/page3" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{{ _('Get the free account that protects your privacy. Join Firefox.') }}{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
+
+{% block body_class %}{{ super() }} welcome-page3{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-3' %}
+{% set _utm_campaign = 'welcome-3-sync' %}
+
+{% block content_intro %}
+  {% call hero(
+    title=_('No account required. But you might want one.'),
+    desc=_('The Firefox browser collects so little data about you, we don’t even require your email address. But when you use it to create a Firefox account, we can protect your privacy across more of your online life.'),
+    class='mzp-t-firefox mzp-t-dark',
+    include_cta=True,
+    heading_level=1,
+    image_url=None,
```

Comment:
Nit: pretty sure you can leave both `image_url` and `include_highres_image` out here if both are falsy?